subjects/git/audit/README.md

```diff
@@ -0,0 +1,185 @@
+
+###### Did the student navigate back and forth between the two tagged versions, `v1` and `v1-beta`?
+
+###### Did the student display a list of all tags present in the repository to verify successful tagging?
+
+#### Changed your mind?
+
+###### Did the student successfully revert the modifications made to the latest version of the file, restoring it to its original state before staging?
+
+###### Did the student introduce unwanted changes to the file, stage them, and then successfully clean the staging area to discard the changes?
+
+###### Did the student add unwanted changes again, stage the file, commit the changes, and then revert them back to their original state?
+
+###### Did the student tag the latest commit with oops and remove commits made after the v1 version, ensuring that the HEAD points to v1?
+
+###### Did the student display the logs with the deleted commits, particularly focusing on the commit tagged `oops`?
+
+###### Did the student ensure that unreferenced commits were deleted from the history, with no logs remaining for these deleted commits?
+
+###### Did the student add author information to the file and commit the changes?
+
+###### Did the student update the file to include the author email without making a new commit, but included the change in the last commit?
+
+#### Move it
+
+###### Did the student successfully move the hello.rb program into a lib/ directory using Git commands?
+
+###### Did the student commit the move of hello.rb?
+
+###### Did the student create and commit a Rakefile in the root directory of the repository with the provided content?
+
+#### blobs, trees and commits
+
+##### Ask the student to navigate to the `.git/` directory and explain to you the purpose of each subdirectory, including `objects/`, `config`, `refs`, and `HEAD`.
+
+###### Was the student able to explain the purpose of each subdirectory, including `objects/`, `config`, `refs`, and `HEAD`?
+
+###### Did the student successfully find the latest object hash within the `.git/objects/` directory using Git commands?
+
+###### Was the student able to print the type and content of this object using Git commands?
+
+###### Did the student use Git commands to dump the directory tree referenced by a specific commit?
+
+###### Were they able to dump the contents of the `lib/` directory and the `hello.rb` file using Git commands?
+
+#### Branching, Merging & Rebasing
+
+###### Did the student successfully create and switch to a new branch named 'greet'?
+
+###### Did the student create and commited a new file named `greeter.rb` in the `lib` directory with the provided Ruby code in it?
+
+###### Did the student update the `lib/hello.rb` file with the provided content, stage, and commit the changes?
+
+###### Did the student update the `lib/Rakefile` with the provided content to ensure it runs the updated `lib/hello.rb` file, stage, and commit the changes?
```

Comment:
As mentioned in a comment in the subject, I feel the `Rakefile` or `Makefile` should be in the root directory and not the `lib` one

Reply:
fixed